src/dvbapi.cpp

```diff
@@ -129,38 +122,19 @@ int dvbapi_reply(sockets *s) {
         int op1;
         b = s->buf + pos;
         copy32r(op, b, 0);
-        op1 = op & 0xFFFFFF;
-        change_endianness = 0;
-        if (op1 == CA_SET_DESCR_X || op1 == CA_SET_DESCR_AES_X ||
-            op1 == CA_SET_PID_X || op1 == DMX_STOP_X ||
-            op1 == DMX_SET_FILTER_X) { // change endianness
-            op = 0x40000000 | ((op1 & 0xFF) << 16) | (op1 & 0xFF00) |
```

Comment:
@catalinii any background on this? Haven't found anything when searching for e.g. CA_SET_DESCR_X nor the corresponding hex values